src/views/settings/Msg/Email/markDownEditor.vue

```diff
@@ -43,6 +43,7 @@ export default {
     },
     localValue(val) {
       this.$emit('input', val)
+      this.$emit('htmlChange', this.html)
     }
   }
 }
```

Comment:
I'm sorry, but the provided code snippet is incomplete. Please share the complete code so I can analyze it for you.